docs/book/src/user/quick-start.md

```diff
@@ -219,6 +291,49 @@ spec:
         cloud-provider: aws
 ```
 {{#/tab }}
+{{#tab Docker}}
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Machine
+metadata:
+  name: capi-quickstart-controlplane-0
+  labels:
+    cluster.x-k8s.io/control-plane: "true"
+    cluster.x-k8s.io/cluster-name: "capi-quickstart"
+    set: controlplane
+spec:
+  version: v1.15.3
+  bootstrap:
+    configRef:
+      apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+      kind: KubeadmConfig
+      name: capi-quickstart-controlplane-0
+  infrastructureRef:
+    kind: DockerMachine
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    name: capi-quickstart-controlplane-0
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: DockerMachine
+metadata:
+  name: capi-quickstart-controlplane-0
+---
+apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+kind: KubeadmConfig
+metadata:
+  name: capi-quickstart-controlplane-0
+spec:
+  initConfiguration:
+    nodeRegistration:
+      kubeletExtraArgs:
+        eviction-hard: nodefs.available<0%,nodefs.inodesFree<0%,imagefs.available<0%
```

Comment:
I don't usually set this, but i haven't been doing much dev work on capd lately -- is this a kind thing?

Reply:
Caught red-handed! I copied this from https://github.com/kubernetes-sigs/cluster-api-provider-docker/blob/master/examples/simple-cluster.yaml, which was updated in https://github.com/kubernetes-sigs/cluster-api-provider-docker/pull/248. :smile: 

That PR doesn't explain these changes, but here's my take:

The lower the thresholds for hard eviction, the fewer total resources are required, and more Pods can be scheduled. WIth these thresholds, Pods can completely starve the node of resources. The default thresholds are higher to prevent this from happening.

Since Docker provider is running on one machine, and isn't intended for production, it might be reasonable to allow scheduling more Pods at the risk of starving the node (a container) of resources.

I'm happy to add an explanation to the doc or manifest.
 
@erwinvaneyk WDYT?

Reply:
To complete the circle, I saw this in @chuckha's https://github.com/chuckha/capi-dev setup. 

Since I ran into eviction issues on my own machine (which had only 20-25% remaining disk space) due to the issue that @dlipovetsky described, I think it is reasonable to add here.

Reply:
perfect. sgtm!

Reply:
Thanks @erwinvaneyk. I'll add an explanatory comment.